packages/api-sdk/src/medical/models/fhir.ts

```diff
@@ -84,9 +84,13 @@ export type ResourceSearchableByPatient = (typeof resourcesSearchableByPatient)[
 export const resourcesSearchableBySubject = ["AdverseEvent", "Task"] as const;
 export type ResourceSearchableBySubject = (typeof resourcesSearchableBySubject)[number];
 
+export const resourcesSearchableByOrganization = ["Organization"] as const;
+export type ResourceSearchableByOrganization = (typeof resourcesSearchableByOrganization)[number];
+
 export const resourceTypeForConsolidation = [
   ...resourcesSearchableByPatient,
   ...resourcesSearchableBySubject,
+  ...resourcesSearchableByOrganization,
```

Comment:
We're not exactly searching by Org here, we're using no filter and searching by tenant.

Perhaps `resourcesSearchableByTenant` would be better?